src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -245,6 +256,21 @@ public AzureAppConfigurationOptions TrimKeyPrefix(string prefix)
             return this;
         }
 
+        /// <summary>
+        /// Trims the provided prefix from the keys of all feature flags retrieved from Azure App Configuration.
+        /// </summary>
+        /// <param name="prefix">The prefix to be trimmed from all feature flags retrieved from Azure App Configuration.</param>
+        public AzureAppConfigurationOptions TrimFeatureFlagPrefix(string prefix)
```

Comment:
>TrimFeatureFlagPrefix [](start = 44, length = 21)

Do we need this method assuming that the existing `TrimKeyPrefix` method works for feature flags too?

Reply:
Same as [this comment](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/234#discussion_r589874260)